File: euler_6/euler_6.js
var sumOfSquares = 0, squareOfSum = 0, answer;

for (i = 1; i < 101; i++) {
    sumOfSquares += i * i;
    squareOfSum += i;
}

squareOfSum = squareOfSum * squareOfSum;

console.log("The answer is: " + (squareOfSum - sumOfSquares));

File: euler_6/euler_6.c
#include <stdio.h>

int main() {

    long sum_of_squares = 0;
    long square_of_sum = 0;
    int i;

    for (i = 1; i < 101; ++i) {
        sum_of_squares += i * i;
        square_of_sum += i;
    }

    square_of_sum = square_of_sum * square_of_sum;

    printf("The answer is: %d\n", square_of_sum - sum_of_squares);

    return 0;
}
